src/views/tables/PositionsTable.tsx

```diff
@@ -302,6 +316,7 @@ const getPositionsTableColumnDef = ({
         columnKey: 'oracle',
         getCellValue: (row) => row.oraclePrice,
         label: stringGetter({ key: STRING_KEYS.ORACLE_PRICE_ABBREVIATED }),
+        isActionable: true,
```

Comment:
instead of adding isActionable to every cell that you want to allow user-select on, I think you should change this in the `<Table>` component. i.e.

within TableCell, you can set isActionable for everything if onRowAction doesn't exist, or even pass a `hasRowAction` boolean into TableCell that will apply the same .stopPropagation that is based on `isActionable`

```
isActionable={
  (
    (cell as GridNode<TableRowData>).column?.value as
      | ColumnDef<TableRowData>
      | undefined
  )?.isActionable ?? onRowAction == null
}
```